### ISSUE TITLE
Documentation Issue: Limit order returns 'orderId' not 'orderOid'

### ISSUE THREAD

**Issue by dafrizzy** [sammchardy/python-kucoin#55]
Minor issue but the comments for the create_limit_order() function state the function returns "orderOid":

.. code:: python
            order = client.create_limit_order('KCS-BTC', Client.SIDE_BUY, '0.01', '1000')
        :returns: ApiResponse
        .. code:: python
            {
                "orderOid": "596186ad07015679730ffa02"
            }
        :raises: KucoinResponseException, KucoinAPIException, LimitOrderException
        """

but after running the code, the buy_response I'm getting appears to be returning "orderId"

**Comment by LeKlex** [comment]
They changed the API response, have a look at https://docs.kucoin.com/#place-a-new-order

**Comment by dafrizzy** [comment]
Yeah that's true it does. They still got rid of "orderOid" though. To be honest, I'm not exactly sure what the difference is between "clientOid", "id", and "orderId" in this new platform for buy/sell/open orders. Open orders seems to return both "clientOid" and "id" while buy and sell limit orders return "orderId". The documentation on their website isn't super clear to me about the differences between each.

**Comment by LeKlex** [comment]
> Yeah that's true it does. They still got rid of "orderOid" though. To be honest, I'm not exactly sure what the difference is between "clientOid", "id", and "orderId" in this new platform for buy/sell/open orders. Open orders seems to return both "clientOid" and "id" while buy and sell limit orders return "orderId". The documentation on their website isn't super clear to me about the differences between each.

The "clientOid" can be set by the client/you (identification of orders in your system), the "id" is given by the KuCoin. You don't need to use the "clientOid", I just go along with KuCoins "id".
